third_party/2and3/redis/client.pyi

```diff
@@ -350,3 +354,9 @@ class Script:
     sha: Any
     def __init__(self, registered_client, script) -> None: ...
     def __call__(self, keys=..., args=..., client=...): ...
+
+class Monitor:
+    def __init__(self, connection_pool) -> None: ...
+    def __enter__(self): ...
+    def __exit__(self, *args): ...
```

Comment:
Standard annotation for `__exit__` is:

```
from types import TracebackType
...
    def __exit__(self, exc_type: Optional[Type[BaseException]], exc_val: Optional[BaseException], exc_tb: Optional[TracebackType]) -> None: ...
```

Reply:
The implementation does not require these parameters, and the docs don't state anything: https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L3020
We can however still use the standard if that's meaningful for this project.

Reply:
It's fine to leave it, but we should at least change the return type to `None` (because that makes a difference), and the `args` type to `Any`.

Reply:
Changed it that way. Thank you!